gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/mapper/UserMapper.java

```diff
@@ -193,6 +198,18 @@ public static io.gravitee.am.model.User convert(User scimUser) {
 
         // set additional information
         if (scimUser instanceof GraviteeUser graviteeUser && graviteeUser.getAdditionalInformation() != null) {
+            var lastPasswordReset = graviteeUser.getAdditionalInformation().get(LAST_PASSWORD_RESET_KEY);
+            if (lastPasswordReset instanceof String) {
+                try {
+                    user.setLastPasswordReset(Date.from(Instant.parse((String) lastPasswordReset)));
+                } catch (Exception e) {
+                    LOGGER.error("Cannot parse lastPasswordReset. Be sure it is in ISO 8601 format.", e);
+                    throw new UserInvalidException("Unable to parse lastPasswordReset date. Be sure it is in ISO 8601 format.", e);
+                }
+            } else {
+                LOGGER.error("lastPasswordReset must be in ISO 8601 format");
+                throw new UserInvalidException("Unable to parse lastPasswordReset. lastPasswordReset must be in ISO 8601 format.");
+            }
```

Comment:
**issue**: lastResetPassword is optional, I'm afraid that if the value is null you will throw an exception.